libgalois/include/katana/Loops.h

```diff
@@ -170,6 +169,10 @@ struct StdForEach {
   }
 };
 
+/**
+ * Helper functor to invoke katana::for_each with the same interface as
+ * katana::do_all
+ */
 struct ForEach {
```

Comment:
The comment is confusing. This provides the same interface as `DoAll`, not `do_all` right?